CMakeLists.txt

```diff
@@ -252,7 +252,3 @@ target_link_libraries(PermutationExporterMain index ${CMAKE_THREAD_LIBS_INIT})
 
 add_executable(VocabularyConverterMain src/VocabularyConverterMain.cpp)
 target_link_libraries(VocabularyConverterMain index ${CMAKE_THREAD_LIBS_INIT})
-
-#add_executable(TextFilterComparison src/experiments/TextFilterComparison.cpp)
-#target_link_libraries (TextFilterComparison experiments)
-
```

Comment:
Why is this commented out? If you want to delete these files completely, make a separate PR out of it.

Reply:
These files don't exist, the comment is dated